rlog.go

```diff
@@ -385,6 +385,17 @@ func Infof(format string, a ...interface{}) {
 	basicLog(levelInfo, notATrace, format, "", a...)
 }
 
+// Println prints a message if RLOG_LEVEL is set to INFO or lower.
```

Comment:
yep will add a comment there.
didn't make it debug because that disables most of the log and we need some fine tuning, till then info is good enough output to have there..